distributed/collections.py

```diff
@@ -109,6 +113,20 @@ def pop(self) -> T:
                 self._data.discard(value)
                 return value
 
+    def topk(self, k: int) -> Iterator[T]:
+        "Iterator over the largest K elements. This is O(1) for k == 1, O(n*logn) otherwise."
```

Comment:
Isn't this yielding the SMALLEST k elements?

Reply:
I find `topk` confusing; in dask.array it means the k biggest elements of a collection.
Could you rename it to `peekn`?

Reply:
Also +1 for having a separate PR